### ISSUE TITLE
Cheatsheet request: Terraform

### ISSUE THREAD

**Issue by somyarj1637** [Fechin/reference#498]
A great piece of work. I want to contribute to it too by giving cheatsheets of Terraform.

**Comment by somyarj1637** [comment]
@Fechin Can I add up the documentation for Terraform?

**Comment by the-ghoulXD** [comment]
If no one is working in it, can I take it up??

**Comment by Fechin** [comment]
[bijuli74](https://github.com/bijuli74) shared https://cheatsheets.zip/terraform. If you're interested, @anuanu0-0, we would love to see any ideas you have based on this.

**Comment by anuanu0-0** [comment]
@Fechin Yes, there are still more things we can add to this like remote backend setups, random, null resources, also all the snippets are around aws, I can help in adding azure related snippets too.

**Comment by Fechin** [comment]
@anuanu0-0 I'm looking forward your PR
